spec/rubocop/options_spec.rb

```diff
@@ -262,4 +262,44 @@ def abs(path)
       end
     end
   end
+
+  describe 'options from file' do
+    let(:command_line_options) { [] }
+
+    before do
+      create_file('.rubocop', %w(--no-color --fail-fast))
+    end
+
+    subject { options.parse(command_line_options).first }
+
+    context 'no command line options given' do
+      it 'reads options from the file' do
+        is_expected.to eq(color: false, fail_fast: true)
+      end
+    end
+
+    context 'command line options given' do
+      let(:command_line_options) { ['--color'] }
+
+      it 'CLI options have priority over file options' do
+        is_expected.to eq(color: true, fail_fast: true)
+      end
+    end
+  end
+
+  describe 'options from RUBOCOP_OPTS environment variable' do
+    let(:command_line_options) { %w(--no-color --fail-fast) }
+
+    around do |example|
+      ENV['RUBOCOP_OPTS'] = '--color'
+      example.run
+      ENV.delete('RUBOCOP_OPTS')
+    end
+
+    subject { options.parse(command_line_options).first }
+
+    it 'ENV options have priority over command line options' do
```

Comment:
I did some research, and have not found any other examples. I also found [a commit](https://github.com/rspec/rspec-core/issues/276) that adds such behaviour to RSpec. And the author just [says](https://github.com/rspec/rspec-core/issues/276#issuecomment-685127) "I suppose both ways are reasonable. SPEC_OPTS priority is better for patching default rspec behaviour."

It doesn't look like this order of precedence is a common practice. So I'd be happy to implement parsing options from environment variables in any order. Having this option is much more flexible when not having it at all.